src/plugins/intel_gpu/src/plugin/program_builder.cpp

```diff
@@ -239,7 +240,11 @@ void ProgramBuilder::CreateSingleLayerPrimitive(const std::shared_ptr<ov::Node>&
             is_created = true;
             break;
         }
-        op_type_info = op_type_info->parent;
+
+        if (std::dynamic_pointer_cast<ov::op::PagedAttentionExtension>(op)) {
+            CreatePagedAttention(*this, op);
```

Comment:
I think now you don't need to have custom path for PA as we can just register impl as usual, right?

Reply:
Oh, I was confused by the omitted op_version, but it could be workaround by introducing an internal version namespace
I moved this PagedAttention op creation logic to a separate file